docs/source/about/contract.rst

```diff
@@ -0,0 +1,117 @@
+===================
+The PaaSTA Contract
+===================
+
+The PaaSTA Contract is similar to the `12 Factor App <http://12factor.net/>`_ documented for Heroku. It specifies what kinds of apps are suitable for PaaSTA and what that app must do to run properly
+
+Basic requirements
+==================
+
+Codebase
+--------
+
+PaaSTA assumes there is one git repo and one resulting Docker image of that service. This image is deployed to multiple locations. This Docker image also may be used for slightly different use cases (worker daemon, web task, etc).
```

Comment:
PaaSTA assumes that the source for a service is stored in a single Git repository, and that Git repository contains a Dockerfile. The image built from that Dockerfile MUST run that service, though the same image MAY support multiple use cases (worker daemon, web task, etc).